### DISCUSSION TITLE
Understanding of TSI results

### DISCUSSION THREAD

**Discussion by cxxuuuuuuu** [davidfrantz/force#160] [Q&A] [answered]
Hi,

I got some problems after getting the interpolated time series data, which means that I don't know which band stands for which date. My time series data start from 2015.08.01 to 2020.12.31 with 16 interpolation days, therefore, there are 123 bands covering the whole time period. Does band1 stand for 2015.08.01, band2 stand for 2015.08.17, band3 stand for 2015.09.02 and so on (every 16 days a date)?

Moreover, I also generated FBW weekly time series data. But there are only 52 weeks. Does it mean just for one year? But I have more than one year ARD data to be used. So I could barely understand what happened. I have read through the FORCE tutorials, but found no answer to my confusion.

I am looking forward to your replies. Thanks:)

Best regards,
Chunyan

**Comment by davidfrantz** [comment, marked as answer]
Hi @cxxuuuuuuu ,

1) the dates are in the metadata in the FORCE domain. When using GDAL, use ``gdalinfo -mdd all`` to display all metadata domains.
2) The ``Fold-by-x`` methods aggregate the time series by the specified period, e.g. Fold-by-week means that all data from all years from week one are aggregated, then all data from all years from week two, and so on. If you want a weekly time series, an interpolation would be the way to go.

Cheers,
David

**Reply by cxxuuuuuuu** [reply]
Hi David,

1. thank you so much for your useful information. By using **gdalinfo -mdd all** I found all the dates which are really as I imagined (16 days one date from 2015.08.01).

2. also thank you for the explanation of Fold-by-week data.

Best regards,
Chunyan
